features/manage_categories.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Manage Categories
+  As a blog administrator
+  In order to organize my thoughts
+  I want to be able to add and edit categories
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Categories List
+    Given I have categories titled Food, Cars
+    When I go to the list of categories
+    Then I should see "Food"
+    And I should see "Cars"
+    When I fill in "category_name" with "Foobar"
```

Comment:
Seems like these are two separate tests. One to see the list and one to see the new category added to the existing list